controllers/ingress/group_controller.go

```diff
@@ -124,7 +128,17 @@ func (r *groupReconciler) reconcile(ctx context.Context, req ctrl.Request) error
 		r.recordIngressGroupEvent(ctx, ingGroup, corev1.EventTypeWarning, k8s.IngressEventReasonFailedAddFinalizer, fmt.Sprintf("Failed add finalizer due to %v", err))
 		return err
 	}
-
+	for _, inactiveMember := range ingGroup.InactiveMembers {
```

Comment:
we could do this check from the model builder task, any reason not to do so?
If we check this from the model builder task, it will be easier to UT. Same comment for service controller as well.

Reply:
Changed accordingly.